backend/config/routes.rb

```diff
@@ -2,6 +2,9 @@
 
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-
+  namespace :api, defaults: { format: :json } do
+    resources :users, only: %i[create show update destroy]
```

Comment:
Remove update for now

Reply:
Okay, have removed update from users' routes.